Exercise_4/docs/part-1.md

```diff
@@ -1,122 +1,306 @@
+    |------------------------------------|----------|--------------------|-----------|---------|-----------|---------|----------|------------------------------------------|
+    |2013-01-05T15:06:33Z                |2013-01-05T15:21:33Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43f55900-14f6-4310-440b-c34398531881|430193006|Medication Reconciliation (procedure)|819.01   |          |                                          |
+    |2014-01-11T15:06:33Z                |2014-01-11T15:17:17Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|171207006|Depression screening (procedure)|600.50   |          |                                          |
+    |2014-01-11T15:06:33Z                |2014-01-11T15:21:33Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|430193006|Medication Reconciliation (procedure)|625.99   |          |                                          |
+    |2014-01-11T15:17:17Z                |2014-01-11T15:37:26Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|715252007|Depression screening using Patient Health Questionnaire Nine Item score (procedure)|43.04    |          |                                          |
+    |2014-01-11T15:37:26Z                |2014-01-11T15:51:39Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|428211000124100|Assessment of substance use (procedure)|600.50   |          |                                          |
+    |2014-01-11T15:51:39Z                |2014-01-11T16:19:37Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|868187001|Assessment using Car  Relax  Alone  Forget  Friends  Trouble Screening Test (procedure)|600.50   |          |                                          |
+    |2014-01-11T16:19:37Z                |2014-01-11T16:33:57Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|386516004|Anticipatory guidance (procedure)|600.50   |          |                                          |
+    |2015-01-17T15:06:33Z                |2015-01-17T15:19:22Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|3dfe4128-1081-13b3-5157-0708939fb2f9|428211000124100|Assessment of substance use (procedure)|600.50   |          |                                          |
+    |2015-01-17T15:19:22Z                |2015-01-17T15:32:51Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|3dfe4128-1081-13b3-5157-0708939fb2f9|868187001|Assessment using Car  Relax  Alone  Forget  Friends  Trouble Screening Test (procedure)|600.50   |          |                                          |
+    |2015-01-17T15:32:51Z                |2015-01-17T15:38:38Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|3dfe4128-1081-13b3-5157-0708939fb2f9|386516004|Anticipatory guidance (procedure)|600.50   |          |                                          |
```

Comment:
```suggestion
??? example "Procedures Table"
      |START                               |STOP      |PATIENT             |ENCOUNTER  |CODE     |DESCRIPTION|REASONCODE|REASONDESCRIPTION                         |
      |------------------------------------|----------|--------------------|-----------|---------|---------|----------|------------------------------------------|
      |2013-01-05T15:06:33Z                |2013-01-05T15:21:33Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43f55900-14f6-4310-440b-c34398531881|430193006|Medication Reconciliation (procedure)|          |                                          |
      |2014-01-11T15:06:33Z                |2014-01-11T15:17:17Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|171207006|Depression screening (procedure)|          |                                          |
      |2014-01-11T15:06:33Z                |2014-01-11T15:21:33Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|430193006|Medication Reconciliation (procedure)|          |                                          |
      |2014-01-11T15:17:17Z                |2014-01-11T15:37:26Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|715252007|Depression screening using Patient Health Questionnaire Nine Item score (procedure)|          |                                          |
      |2014-01-11T15:37:26Z                |2014-01-11T15:51:39Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|428211000124100|Assessment of substance use (procedure)|         |                                          |
      |2014-01-11T15:51:39Z                |2014-01-11T16:19:37Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|868187001|Assessment using Car  Relax  Alone  Forget  Friends  Trouble Screening Test (procedure)|          |                                          |
      |2014-01-11T16:19:37Z                |2014-01-11T16:33:57Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|43756071-53a9-c553-9e53-d5579d869a59|386516004|Anticipatory guidance (procedure)|         |                                          |
      |2015-01-17T15:06:33Z                |2015-01-17T15:19:22Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|3dfe4128-1081-13b3-5157-0708939fb2f9|428211000124100|Assessment of substance use (procedure)|         |                                          |
      |2015-01-17T15:19:22Z                |2015-01-17T15:32:51Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|3dfe4128-1081-13b3-5157-0708939fb2f9|868187001|Assessment using Car  Relax  Alone  Forget  Friends  Trouble Screening Test (procedure)|          |                                          |
      |2015-01-17T15:32:51Z                |2015-01-17T15:38:38Z|38b9dfb6-29f6-60c8-ef37-6210064feac0|3dfe4128-1081-13b3-5157-0708939fb2f9|386516004|Anticipatory guidance (procedure)|         |                                          |
```

Don't include the costs... they also are not typically in a patient's clinical data.